### ISSUE TITLE
hack/local-up-cluster.sh cannot find etcd

### ISSUE THREAD

**Issue by arun-gupta** [ghodss/kubernetes-macosx-development#2]
Running hack/local-up-cluster.sh cannot find etcd and gives the following error:

```
+++ Placing binaries
Starting etcd
which: no etcd in (/usr/local/bin:/usr/bin:/usr/local/sbin:/usr/sbin:/home/vagrant/gopath/bin:/home/vagrant/.local/bin:/home/vagrant/bin)

etcd must be in your PATH
```

**Comment by arun-gupta** [comment]
`vagrant up` gave the following error:

==> k8s-env: Installing godep and etcd...
==> k8s-env: sudo: sorry, you must have a tty to run sudo
==> k8s-env: sudo: sorry, you must have a tty to run sudo

And this might be the reason etcd was not installed. Any suggestions on how to fix it?

**Comment by arun-gupta** [comment]
Worked around this by adding `sed -i 's/requiretty/\!requiretty/g' /etc/sudoers` to `setup.sh`.

**Comment by arun-gupta** [comment]
Sent PR https://github.com/ghodss/kubernetes-macosx-development/pull/4

**Comment by arun-gupta** [comment]
Apparently PR #4 only solves part of the problem, #3 would need to be fixed before this can be resolved.

**Comment by ghodss** [comment]
TTY issue is resolved by #4, passing GOPATH into sudo is solved by https://github.com/ghodss/kubernetes-macosx-development/commit/491cd022b7298ec313899812df70c01778f5ca17. etcd should now install and work correctly.
